hack/windows/Install-OVS.ps1

```diff
@@ -199,6 +199,22 @@ function InstallOVS() {
     SetEnvVar "Path" $env:Path
 }
 
+function CheckIfLibInstalled() {
+    # Check if SSL library has been installed
+    $pathEntries = $env:Path.Split(";") | ForEach-Object {
+        if ((Test-Path "$_/ssleay32.dll" -PathType Leaf) -and (Test-Path "$_/libeay32.dll" -PathType Leaf)) {
+            Log "Found existing SSL library."
+            return
+        }
+    }
+    $SSLZip = "openssl-1.0.2u-x64_86-win64.zip"
+    curl.exe -LO https://indy.fulgan.com/SSL/$SSLZip
```

Comment:
Add check on the curl result before trying to unzip it?

Reply:
Updated.